### DISCUSSION TITLE
A way to access subscription topic in realtime-api-hooks?

### DISCUSSION THREAD

**Discussion by youssefsiam38** [pocketbase/pocketbase#4298] [Q&A] [answered]
Is these is a way to access subscription topic from inside goalng event hook

for example...
```go
	app.OnRealtimeConnectRequest().Add(func(e *core.RealtimeConnectEvent) error {
		log.Println("RealtimeConnectRequest")
		fmt.Printf("topic: %+v\n", e.Client.Get("topic"))
		return nil
	})
```

**Comment by ganigeorgiev** [comment, marked as answer]
`OnRealtimeConnectRequest` is fired when the client initialize the connection. The subscriptions happens at later stage (eg. on `OnRealtimeAfterSubscribeRequest`).

You can access the client subscriptions with `e.Client.Subscriptions()`.

If you need further help, please elaborate what you are trying to do.

**Comment by Salman9000** [comment]


**Reply by ganigeorgiev** [reply]
Please don't spam!

**Reply by Salman9000** [reply]
Hi, I am not sure what happened, but my comment is not being displayed. It was not my intention to spam.

**Reply by ganigeorgiev** [reply]
@Salman9000 If you have a question, feel free to open a new Q&A discussion but try to elaborate more clearly on what actually is the problem and what you are trying to do.

Hijacking old discussions and saying things like "X doesn't work" without any details what is X, how it is related to the discussion where you posted it and what is even the expected behavior I consider to be lazy and low effort (to make the matter worst it makes GitHub search unusuable for everybody).
